emcc.py

```diff
@@ -1854,12 +1854,12 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
       src = open(final).read()
       final = final + '.modular.js'
       f = open(final, 'w')
-      f.write('var ' + shared.Settings.EXPORT_NAME + ' = function(Module) {\n')
-      f.write('  Module = Module || {};\n')
+      f.write('var ' + shared.Settings.EXPORT_NAME + ' = function(' + shared.Settings.EXPORT_NAME + ') {\n')
+      f.write('  ' + shared.Settings.EXPORT_NAME + ' = ' + shared.Settings.EXPORT_NAME + ' || {};\n')
       f.write('\n')
       f.write(src)
       f.write('\n')
-      f.write('  return Module;\n')
+      f.write('  return ' + shared.Settings.EXPORT_NAME + ';\n')
```

Comment:
This could also be as it was before, since the code in the middle will assign the exported name to `Module`. So I guess it's a matter of which is easier to read. I think in this file it's easier your way, but in the generated code where there can be a lot of code in the middle, it's better the other way. I think the second case is more common, so I'd prefer to return `Module` here.